contrib/occtax/frontend/app/occtax-form/occurrence/occurrence.service.ts

```diff
@@ -162,6 +185,203 @@ export class OcctaxFormOccurrenceService {
         })
       )
       .subscribe((val) => this.form.get("non_digital_proof").setValue(val));
+
+      
+    /* MET Champs additionnel, récupérer le dataset */
+    this.occtaxFormService.editionMode
+      .pipe(
+        switchMap((editionMode: boolean) => {
+          //Le switch permet, selon si édition ou creation, de récuperer les valeur par defaut ou celle de l'API
+          return editionMode ? this.releveValues : [];
+        })
+      )
+      .subscribe((values) => this.data = values); 
```

Comment:
A quoi sert le `this.data` ?
Il a l'air inutilisé

Reply:
Il est inutilisé.